johnsonandjohnson/Bodiless-JS
# fix(ui): Adjust contextual menu position

## Changes
Adjust contextual menu position, set it to bottom-left.

## Test Instructions
- Visit /filter-page /list /products and try to click around in edit mode.
- The contextual menu should appear at the bottom-left part of the edit block and be consistent for all components (image, link, menu, list).

## Related Issues
Closes https://github.com/johnsonandjohnson/Bodiless-JS/issues/318
